lib/spack/docs/getting_started.rst

```diff
@@ -283,7 +283,7 @@ valid flags are ``cflags``, ``cxxflags``, ``fflags``, ``cppflags``,
          cxx: /usr/local/bin/icpc-15.0.024-beta
          f77: /usr/local/bin/ifort-15.0.024-beta
          fc: /usr/local/bin/ifort-15.0.024-beta
-       parameters:
+       flags
```

Comment:
Sorry for the delay, yes, there should be a colon here. Can you please add it in there? I'm having some git issues.